src/RuleSet.php

```diff
@@ -174,7 +175,6 @@ final class RuleSet implements RuleSetInterface
             'ereg_to_preg' => true,
             'error_suppression' => true,
             'fopen_flag_order' => true,
-            'fopen_flags' => ['b_mode' => false],
```

Comment:
https://github.com/FriendsOfPHP/PHP-CS-Fixer/issues/4011 I think this should stay, it was disabled before in the symfony repo because it would add the `b`, now it will remove it (when we ship the next version of the tool that is)

Reply:
done